kong/plugins/log-serializers/runscope.lua

```diff
@@ -0,0 +1,41 @@
+local _M = {}
+
+function _M.serialize(ngx)
+  
+  -- JSON format below based on Runscope Messages resource:
+  -- https://www.runscope.com/docs/api/messages#message-create
+  -- Not sent: req/res:body_encoding, req:form, res:reason
```

Comment:
Any reason for not including `request.form` since the form parameters are available in `ngx.ctx.runscope.req_post_args`?